deploy/kubectl_helpers/utils/copy.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+DIR=$(dirname "$BASH_SOURCE")
+
+set -x -e
+
+DEPLOYMENT_TARGET=$1
+COMPONENT=$2
+FILE=$3
+TO_POD=$4
```

Comment:
is TO_POD expected to be a boolean here? It might be worth adding a comment to explain this a little bit (particularly since I think the default is going to end up true if you don't pass a $4, which could be a little confusing).